schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -575,6 +575,40 @@ private static void importTables(Either<Path, Properties> config, Either<Path, S
     }
   }
 
+  /**
+   * Upgrades the ScalarDB environment to support the latest version of the ScalarDB API. Typically,
+   * you will be requested, as indicated on the release notes, to run this method after updating the
+   * ScalarDB version of your application environment.
+   *
+   * @param configPath path to the ScalarDB config properties.
+   * @param options specific options for upgrading.
+   * @throws SchemaLoaderException thrown when upgrading failed.
+   */
+  public static void upgrade(Path configPath, Map<String, String> options)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Left<>(configPath);
+    upgrade(config, options);
+  }
```

Comment:
Super minor, but we probably need an empty line?

Reply:
Indeed, thank you.
It's a shame Spotless does not enforce a blank line between method definitions.
https://github.com/scalar-labs/scalardb/pull/1208/commits/be5165f9652744402a98758aa98ecf7320b26d7f